source/frontend/src/hooks/useRefreshSiteminder.ts

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import { useCallback, useMemo } from 'react';
+
+import { useTenant } from '@/tenants';
+
+export const useRefreshSiteminder = () => {
+  const { parcelsLayerUrl, siteminderRefreshUrl } = useTenant();
+
+  const refresh = useCallback(() => {
+    axios.get(parcelsLayerUrl + siteminderRefreshUrl);
+  }, [parcelsLayerUrl, siteminderRefreshUrl]);
+
+  return useMemo(
```

Comment:
this looks kind odd, but should it be a state rather than a computed variable that only depends on itself for updating?

Reply:
uh, yeah, this is from a pattern I use where multiple functions are being returned from a hook. Not necessary in this case.